src/ai-function/ai-runner.ts

```diff
@@ -14,29 +15,29 @@ export function createAIRunner<Content = string>(args: {
   /** The ChatModel used to make API calls. */
   chatModel: Model.Chat.Model;
   /** The functions the model can call. */
-  functions?: Prompt.AIFunction[];
+  functions?: AIFunction[];
   /** Use this to control when the runner should stop. */
-  shouldBreakLoop?: (msg: Prompt.Msg) => boolean;
+  shouldBreakLoop?: (msg: Msg) => boolean;
   /** The maximum number of iterations before the runner throws an error. An iteration is a single call to the model/API. */
   maxIterations?: number;
   /** The number of function calls to make concurrently. */
   functionCallConcurrency?: number;
   /** Parse and validate the content of the last message. */
   validateContent?: (content: string | null) => Content | Promise<Content>;
   /** Controls whether functions or tool_calls are used. */
-  mode?: Prompt.Runner.Mode;
+  mode?: Runner.Mode;
   /** Add a system message to the beginning of the messages array. */
   systemMessage?: string;
   /** Called when a retriable error occurs. */
   onRetriableError?: (error: Error) => void;
-}): Prompt.Runner<Content> {
+}): Runner<Content> {
```

Comment:
`createAIFunction` => `AIFunction`
`createAIRunner` => `Runner`

(nit) seems wrong to me but if we're only special-casing the `AI` prefix for `AIFunction` because otherwise it would mask a built-in `Function`, then it's fine; I'd just rather be consistent everywhere if possible.